volatility3/framework/symbols/linux/__init__.py

```diff
@@ -348,3 +375,30 @@ def container_of(
         return vmlinux.object(
             object_type=type_name, offset=container_addr, absolute=True
         )
+
+    @classmethod
+    def get_vmlinux_from_volobj(cls, volobj):
```

Comment:
I understand the intention, but this isn't accurately named.  The return value (with a type annotation if possible), appears to be the module the object's type belongs to.  At the moment, we don't really use application specific symbol tables, but the hope is we will one day, and so for a piece of the core API, this really needs to be named properly from the get go.